File: Backend-BEPlateform/src/main/java/com/example/backendbeplateform/DAO/Entities/DecisionProduitExtends/Maitrise.java
package com.example.backendbeplateform.DAO.Entities.DecisionProduitExtends;


import com.example.backendbeplateform.DAO.Entities.DecisionProduit;
import lombok.*;

import javax.persistence.*;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
@ToString
@Entity
public class Maitrise extends DecisionProduit {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name="idMaitrise")
    private int idMaitrise;

    private Boolean fencTransmettre;

    private String atelier;

    private Boolean inspectionRecord;

    private Boolean planSurveillance;

    private Boolean resolutionProbleme;

    private Boolean aucunSuivi;

    private Boolean derogationProduit;

    private String reccurenceFenc;
}

File: ngx-admin/ngx-admin/src/app/@core/Models/DecisionProduitExtends/Maitrise.ts
import { DecisionProduit } from "../DecisionProduit";

export class Maitrise extends DecisionProduit{
    
    idMaitrise:number
    fencTransmettre:Boolean
    atelier:string
    inspectionRecord:Boolean
    planSurveillance:Boolean
    resolutionProbleme:Boolean
    aucunSuivi:Boolean
    derogationProduitLivrable:Boolean


}